.github/workflows/tests.yaml

```diff
@@ -25,6 +25,5 @@ jobs:
       run: make PREFIX=artifacts cmds
     - name: List binaries
       run: ls -al artifacts/
-    # no tests yet
-    # - name: Test
-    #   run: go test -v -race ./...
+    - name: Test
+      run: go test -v -race ./...
```

Comment:
I also made this particular change in #70, but added that again here to run the new tests added by this PR in CI.